doc/vm-creation-example.md

```diff
@@ -0,0 +1,103 @@
+# Example - Allocating MAC Addresses to a VirtualMachine
+
+## Create a network-attachment-definition:
+
+The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the VirtualMachine.
```

Comment:
Mind dropping quotes? You haven't used them for VirtualMachine

Reply:
Sure